docs/specifying-a-call-to-configure.md

```diff
@@ -55,6 +55,9 @@ A.CallTo(fakeShop).Throws(new Exception());
 // Or limit the calls by return type
 A.CallTo(fakeShop).WithReturnType<string>().Returns("sugar tastes good");
 
+// Or limit the calls to methods that return a value
+A.CallTo(fakeShop).WithNonVoidReturnType().Returns("sugar tastes good");
+

```

Comment:
Interesting idea. Me, I like to put the scary powerful things at the bottom, in the  hopes that people stop reading before they found it, because they found the safe thing that's probably what the really want.

`ReturnsLazily` is less restrictive than `Returns`, but I'd rather talk about `Returns` first.